packages/twenty-front/src/modules/ui/input/editor/utils/getFirstNonEmptyLineOfRichText.ts

```diff
@@ -0,0 +1,23 @@
+import { PartialBlock } from '@blocknote/core';
+import { isUndefinedOrNull } from '~/utils/isUndefinedOrNull';
+
+export const getFirstNonEmptyLineOfRichText = (
```

Comment:
I wonder if this util should be part of ui/input/editor or should be tied to the field type ui/fields/display or ui/fields/input ? No strong opinion here. It's a bit similar to our utils to parse urls for example

Reply:
I get your point but I feel it's better here because it's highly coupled with Blocknote and that's where most of  Blocknote-related code lives + we might want to use it in places that are not related to fields/display